packages/react-paypal-js/src/v6/hooks/usePayPalGuestPaymentSession.ts

```diff
@@ -0,0 +1,140 @@
+import { useCallback, useEffect, useRef } from "react";
+
+import { usePayPal } from "./usePayPal";
+import { useIsMountedRef } from "./useIsMounted";
+import { useError } from "./useError";
+import { useProxyProps } from "../utils";
+
+import type {
+    PayPalGuestOneTimePaymentSession,
+    PayPalGuestOneTimePaymentSessionOptions,
+    PayPalGuestOneTimePaymentSessionPromise,
+    PayPalGuestPresentationModeOptions,
+} from "@paypal/paypal-js/sdk-v6";
+import type { BasePaymentSessionReturn } from "../types";
+
+export interface PayPalGuestPaymentSessionReturn
```

Comment:
Does this interface need to be exported? I don't see it used anywhere outside this file.